ts/src/sdk/transport/tx/createStargateClient/createGenericStargateClient.spec.ts

```diff
@@ -109,7 +109,7 @@ describe(createGenericStargateClient.name, () => {
         getMessageType,
         createClient: async () => ({
           sign: jest.fn(),
-          simulate: jest.fn(),
+          simulate: jest.fn(async () => 1),
```

Comment:
Just a very small detail for readability but could we have this asynchronous function as a constant that ca be passed to simulating with the name that makes the behavior clear? Something more readable than `async () => 1`

Reply:
I understand, my suggestion would be `jest.fn(MOCK_SIMULATE)`